app/version.go

```diff
@@ -3,7 +3,7 @@ package app
 import "github.com/urfave/cli/v2"
 
 const (
-	DefaultVersion = "v0.5.0"
+	DefaultVersion = "v0.6.0"
```

Comment:
I do wonder if we should cut releases (bump the version number) in a separate PR. It helps keep the commit history clean and easy to understand at a glance.